proxy-node-shared/src/main/java/uk/gov/ida/notification/contracts/HubResponseTranslatorRequest.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.ida.notification.contracts;
```

Comment:
If this is a dto, should it go in the same module as the others - `uk.gov.ida.notification.dto`?

Reply:
Renamed to `uk.gov.ida.notification.contracts`